WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -253,6 +253,10 @@ class MeViewController: UITableViewController {
 
     private func pushAbout() -> ImmuTableAction {
         return { [unowned self] _ in
+            defer {
+                self.tableView.deselectSelectedRowWithAnimation(true)
+            }
```

Comment:
Why not execute this directly at the bottom of this code block to avoiding the extra `defer`  here?

Reply:
I was just replicating the method below, but I think I prefer it in the completion block for the presentation. 👍